assemble/bin/accumulo-service

```diff
@@ -137,40 +192,61 @@ function main() {
   mkdir -p "$ACCUMULO_LOG_DIR" 2>/dev/null
   mkdir -p "$ACCUMULO_PID_DIR" 2>/dev/null
 
-  host="$(hostname)"
-  if [[ -z $host ]]; then
-    host=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')
+  HOST="$(hostname)"
+  if [[ -z $HOST ]]; then
+    HOST=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')
   fi
-  service="$1"
 
-  if [[ $service == "master" ]]; then
+  local service_type="$1"
+  local command_name="$2"
+  shift 2
```

Comment:
No change needed here for this comment... I'm just making an observation.

I don't remember ever using the numeric option on shift before. I probably would have written this as:

```bash
  local service_type="$1"
  local command_name="$2"
  shift; shift
```

Or even:

```bash
  local service_type="$1"; shift
  local command_name="$1"; shift
```

I wonder if this is a newer feature of bash or has always been there and I just never noticed.

Reply:
Interesting, I think I've always used the numeric option.